### DISCUSSION TITLE
Github Page doesn't show the posts

### DISCUSSION THREAD

**Discussion by Northbreeze** [community/community#22756] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I’ve followed the lab instructions to create a new GitHub page . I could successfully create the welcome page; however, my posts won’t appear on the site. For instance, I’ve one post under _posts folder per below but it doesn’t show on the site: <a href="https://github.com/Northbreeze/Nasimt/blob/master/_posts/2020-09-02-aboutme.md" rel="nofollow noopener">https://github.com/Northbreeze/Nasimt/blob/master/_posts/2020-09-02-aboutme.md</a>. What is the problem?</p>
<p>Here is my site: <a href="https://northbreeze.github.io/Nasimt/" rel="nofollow noopener">https://northbreeze.github.io/Nasimt/</a></p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/northbreeze">@Northbreeze</a>. This is happening because the template you’ve chosen doesn’t include a blog section in the default page layout. You’ll either need to choose a different theme that does include a blog section on the homepage, or modify this theme’s template to include a layout that lists your blog posts.</p>
<p>If you’d like to change theme I believe the only default Pages theme that supports a blog format is the minima theme:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/jekyll/minima" target="_blank" rel="noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2439407/181073028-9f63629a-cae9-486e-a8c1-2ef229ea72cb.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/jekyll/minima" target="_blank" rel="noopener">jekyll/minima</a></h3>

<p>Minima is a one-size-fits-all Jekyll theme for writers. - jekyll/minima</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>You can however choose any other gem-based Jekyll theme hosted on GitHub that has a blog layout and use it with the <code>remote_theme</code> variable:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll#adding-a-theme" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll#adding-a-theme" target="_blank" rel="noopener">Adding a theme to your GitHub Pages site using Jekyll - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If you’d prefer to keep your current theme you’ll need to edit or add HTML layouts to include a blog section. You can see how to do this here:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll#customizing-your-themes-html-layout" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll#customizing-your-themes-html-layout" target="_blank" rel="noopener">Adding a theme to your GitHub Pages site using Jekyll - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
